src/doc/trpl/error-handling.md

```diff
@@ -1,6 +1,6 @@
 % Error Handling
 
-> The best-laid plans of mice and men  
```

Comment:
These spaces are correct – they are needed to force the soft line break in markdown (`<br/>`).

Please make sure the other markdown files don't contain spaces like these before removing them.